The beta pipeline was mostly great. It ran into 

```
Caused by: /home/jazz_build/Build/jbe/build/WS-CD-Open/dev/ant_build/public_imports/internal_imports/delivery_validation.xml:271: Could not identify any suitable builds to compare to on the definition Open Liberty Personal Build CI Orchestrator - EBC
```

during an Ant task flow that's really an obsoleted leftover from days past (revertChangesets custom task); this failed because the underlying code was looking for previous builds of that type and since this is the first one requested, it couldn't find any other and thus failed. And then the packaging verification build failed due to a problem with libertyfs:

```
[15-Mar-2023 02:50:18:103 PDT] 00000050 BUILD FAILED
[15-Mar-2023 02:50:18:103 PDT] 00000050 /home/jazz_build/Build/jbe/build/dev/ant_build/build-packaging-verification.xml:132: Encountered an exception while searching for matching files. Directory was: /liberty/dev/Xo/release/cl230220230222-1257-_Iaaa0LKoEe2syNb0fJgimw
```

This was discussed at https://ibm-cloud.slack.com/archives/C30NGUQ9E/p1678879575805539.

Anyway, I'm re-requesting more.

--- a/.ci-orchestrator/compilePersonal.properties
+++ b/.ci-orchestrator/compilePersonal.properties
@@ -21,7 +21,7 @@ ebc.shortlist=parentbuild-personal.yml
 enable.predelivery.checking=true
 fail.gradle.on.error=false
 # Simple ant exclude pattern, takes only a single value
-fat.buckets.to.exclude=com.ibm.ws.ui*fat.gui*,com.ibm.ws.os.packaging_fat
+fat.buckets.to.exclude=
 # all, auto, none, or a comma-separated list of FAT project names.
 fat.buckets.to.run=auto
 # Do not change, if you need to run full fat buckets please use the spawn.fullfat.buckets property instead

